src/components/RadioButton/RadioButton.tsx

```diff
@@ -50,12 +50,24 @@ export function RadioButton({
   const id = useUniqueId('RadioButton', idProp);
   const name = nameProp || id;
   const {unstableGlobalTheming = false} = useFeatures();
+  const inputNode = useRef<HTMLInputElement | null>(null);
```

Comment:
```suggestion
  const inputNode = useRef<HTMLInputElement>(null);
```

I think should just work